The complex gradient checks fail in PPC for some reason, so skip those.
See https://github.com/pytorch/pytorch/issues/57617

Author: Alexander Grund (TU Dresden)

diff --git a/torch/testing/_internal/common_methods_invocations.py b/torch/testing/_internal/common_methods_invocations.py
index 101a86c8d6..dfd80568e8 100644
--- a/torch/testing/_internal/common_methods_invocations.py
+++ b/torch/testing/_internal/common_methods_invocations.py
@@ -25,7 +25,7 @@ from torch.testing._internal.common_utils import \
      random_symmetric_matrix, random_symmetric_psd_matrix,
      random_symmetric_pd_matrix, make_nonzero_det,
      random_fullrank_matrix_distinct_singular_value, set_rng_seed,
-     TEST_WITH_ROCM, IS_WINDOWS, IS_MACOS, make_tensor, TEST_SCIPY,
+     TEST_WITH_ROCM, IS_WINDOWS, IS_MACOS, IS_PPC, make_tensor, TEST_SCIPY,
      torch_to_numpy_dtype_dict, slowTest, TEST_WITH_ASAN)
 
 from distutils.version import LooseVersion
@@ -1967,6 +1967,12 @@ op_db: List[OpInfo] = [
                    dtypesIfROCM=all_types_and_complex_and(torch.bool),
                    decorators=(precisionOverride({torch.float16: 1e-2,
                                                   torch.bfloat16: 1e-2}),),
+                   skips=(
+                       # Complex grad check fails on PPC
+                       SkipInfo('TestGradients', 'test_fn_grad',
+                        device_type='cpu', dtypes=[torch.complex128],
+                        active_if=IS_PPC),
+                   ),
                    safe_casts_outputs=True,
                    supports_complex_to_float=True,
                    test_inplace_grad=False),
